packages/instantsearch.js/scripts/rollup/rollup.config.js

```diff
@@ -25,6 +25,7 @@ const plugins = [
     rootMode: 'upward',
     exclude: /node_modules/,
     extensions: ['.js', '.ts', '.tsx'],
+    sourceType: 'unambiguous',
```

Comment:
```suggestion
    exclude: /node_modules|algoliasearch-helper/,
    extensions: ['.js', '.ts', '.tsx'],
```

this works for me @dhayab, it leaves commonjs in place for the commonjs plugin, and doesn't add a bunch of babel plugins, essentially still using the helper as an external dependency.


I'm surprised to why the change is only done in InstantSearch.js, probably needed for React InstantSearch Hooks, React InstantSearch Core and Vue InstantSearch too right?

Reply:
Awesome, that seems to work indeed! As for React InstantSearch and Vue, we might have the same issue, it's probably that it's less common to have UMD implementations of those, maybe?